### PR TITLE
add polkachu endpoints

### PR DESCRIPTION
**Pull request by PolkachuIntern**
## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: docs/nodes/mocha-testnet.mdx [hunk 1]
```diff
@@ -55,6 +55,7 @@ Below is a list of RPC endpoints you can use to connect to Mocha testnet:
 
 * `rpc-mocha.pops.one`
 * `rpc.mocha.celestia.counterpoint.software`
+* [https://celestia-testnet-rpc.polkachu.com](https://celestia-testnet-rpc.polkachu.com)
```
Comment (jcstein) [inline docs/nodes/mocha-testnet.mdx:58]: is this expected?

https://celestia-testnet-rpc.polkachu.com/block?height=_

```json
{
"jsonrpc": "2.0",
"id": -1,
"error": {
"code": -32602,
"message": "Invalid params",
"data": "error converting http params to arguments: invalid 64-bit integer encoding \"_\", expected string"
}
}
```

Comment (jcstein) [reply docs/nodes/mocha-testnet.mdx:58]: please format this the same as others, as well, thanks!

**Comment by PolkachuIntern** [comment]
Sorry. Will close this PR and submit a new one. Thanks,

**Comment by jcstein** [comment]
no problem! sounds good

File: docs/nodes/mocha-testnet.mdx (hunk 1 above)
Comment (PolkachuIntern) [reply docs/nodes/mocha-testnet.mdx:58]: Just want to reply to two questions.
1. You need to specify a block number if you want to query a block. For example, https://celestia-testnet-rpc.polkachu.com/block?height=65420 It is expected to receive an error when you do not provide a block number. 
2. A new PR is submitted to fix the format issue. Thanks.

Comment (jcstein) [reply docs/nodes/mocha-testnet.mdx:58]: :smile: thank you for the context!

Comment (jcstein) [reply docs/nodes/mocha-testnet.mdx:58]: 🤦

**Pull request closed without merging**